<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Orecchiette</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="audioSourceLabel">
        <property name="text">
         <string>Audio Source</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="audioComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Microphone</string>
         </property>
         <property name="icon">
          <iconset resource="OreResources.qrc">
           <normaloff>:/icons/mic.png</normaloff>:/icons/mic.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Speaker</string>
         </property>
         <property name="icon">
          <iconset resource="OreResources.qrc">
           <normaloff>:/icons/speaker.png</normaloff>:/icons/speaker.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mic and Speaker</string>
         </property>
         <property name="icon">
          <iconset resource="OreResources.qrc">
           <normaloff>:/icons/phone.png</normaloff>:/icons/phone.png</iconset>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="videoSourceLabel">
        <property name="text">
         <string>Video Source</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="videoComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Screen</string>
         </property>
         <property name="icon">
          <iconset resource="OreResources.qrc">
           <normaloff>:/icons/monitor.png</normaloff>:/icons/monitor.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Rear Camera</string>
         </property>
         <property name="icon">
          <iconset resource="OreResources.qrc">
           <normaloff>:/icons/camera.png</normaloff>:/icons/camera.png</iconset>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Front Camera</string>
         </property>
         <property name="icon">
          <iconset resource="OreResources.qrc">
           <normaloff>:/icons/user.png</normaloff>:/icons/user.png</iconset>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Status: Idle</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="recordButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Record</string>
      </property>
      <property name="icon">
       <iconset resource="OreResources.qrc">
        <normaloff>:/icons/notrecording.png</normaloff>:/icons/notrecording.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>48</width>
        <height>48</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="playButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Play</string>
        </property>
        <property name="icon">
         <iconset resource="OreResources.qrc">
          <normaloff>:/icons/play.png</normaloff>:/icons/play.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pauseButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Pause</string>
        </property>
        <property name="icon">
         <iconset resource="OreResources.qrc">
          <normaloff>:/icons/pause.png</normaloff>:/icons/pause.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stopButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
        <property name="icon">
         <iconset resource="OreResources.qrc">
          <normaloff>:/icons/stop.png</normaloff>:/icons/stop.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOrecchiette">
    <property name="title">
     <string>Orecchiette</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="actionDocumentation"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuOrecchiette"/>
  </widget>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="text">
    <string>Documentation</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="OreResources.qrc"/>
 </resources>
 <connections/>
</ui>
